__tests__/kata7.getEmployerRole.test.js

```diff
@@ -1,7 +1,19 @@
-const { getEmployerRole } = require('../src');
+const getEmployerRole = require('../src/kata7.getEmployerRole.js');
+const employees = [{
+  name: 'Satti',
+  role: 'Developer'
+}, {
+  name: 'Jenny',
+  role: 'Sales Associate'
+}, {
+  name: 'Javid',
+  role: 'Human Recommended Reading Assistant'
+}]
 
 describe('getEmployerRole', () => {
   it('returns the employee\'s role in the company', () => {
-    
+    const employeeName = 'Jenny'
+    const employeeRole = getEmployerRole(employeeName, employees)
```

Comment:
Is this line necessary? It doesn't look like you're doing anything with the `const` after you define it.